template/.github/workflows/ci.yml

```diff
@@ -19,8 +19,8 @@ jobs:
       - name: Get Python version for other CI jobs
         id: vars
         run: |
-          echo "min_python=$(cat .github/workflows/python-version-ci)" >> $GITHUB_OUTPUT
-          echo "min_tox_env=py$(cat .github/workflows/python-version-ci | sed 's/\.//g')" >> $GITHUB_OUTPUT
+          echo "min_python=$(< .github/workflows/python-version-ci)" >> "$GITHUB_OUTPUT"
+          echo "min_tox_env=py$(sed 's/\.//g' < .github/workflows/python-version-ci)" >> "$GITHUB_OUTPUT"
```

Comment:
This avoid [Useless use of cat](https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat)